faces.py

```diff
@@ -111,11 +111,27 @@ def get_random_person_from_all_batches(cursor, current_user):
                              last_name,
                              image_url
                       FROM people
-                      WHERE person_id != %s
+                      WHERE person_id != %s AND NOT image_url LIKE '%%no_photo_%%'
```

Comment:
Tiny formatting thing - would you mind splitting the additional `WHERE` clauses onto separate lines? So like in this case:

```suggestion
                      WHERE person_id != %s
                        AND NOT image_url LIKE '%%no_photo_%%'
```